qctrlopencontrols/driven_controls/__init__.py

```diff
@@ -31,12 +31,11 @@
     SCROFULOUS,
     CORPSE_IN_SCROFULOUS)
 
-from .predefined import (
-    new_predefined_driven_control,
-    new_primitive_control, new_wimperis_1_control, new_solovay_kitaev_1_control,
-    new_compensating_for_off_resonance_with_a_pulse_sequence_control,
-    new_compensating_for_off_resonance_with_a_pulse_sequence_with_solovay_kitaev_control,
-    new_compensating_for_off_resonance_with_a_pulse_sequence_with_wimperis_control,
-    new_short_composite_rotation_for_undoing_length_over_and_under_shoot_control,
-    new_walsh_amplitude_modulated_filter_1_control,
-    new_corpse_in_scrofulous_control)
+from .predefined import new_predefined_driven_control
+
+__all__ = ['UPPER_BOUND_RABI_RATE', 'UPPER_BOUND_DETUNING_RATE',
```

Comment:
IMO it'd be best to have one item per line, in alphabetical order. Makes scanning the code a little easier.

Reply:
Updated; Made the list with one liberty [ALL CONSTANTS, AllClasses, all_methods] within each, alphabetical order in maintained.